receiver_custom.html

```diff
@@ -403,7 +406,24 @@ <h3 id= "error_description" class ="cast_text"></h3>
       // prepare the player load script
       var playerJs = document.createElement("script");
       playerJs.type = "text/javascript";
-      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
+      playerJs.src = OOYALA_PLAYER_URL;
+      if (debug || v3Version) {
+        if (!playerJs.src.match(/\?$/)) {
+          playerJs.src += "?";
+        } else if (playerJs.src.match(/=/)) {
```

Comment:
is = sign allowed before ? in URL?

On Tuesday, January 26, 2016, aldigun <notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:

> In receiver_custom.html
> <https://github.com/ooyala/chromecast-sample-receiver/pull/35#discussion_r50929385>
> :
>
> > @@ -403,7 +406,24 @@ <h3 id= "error_description" class ="cast_text"></h3>
> >        // prepare the player load script
> >        var playerJs = document.createElement("script");
> >        playerJs.type = "text/javascript";
> > -      playerJs.src = OOYALA_PLAYER_URL + "?" + (debug? ("debug=true&" + v3Version) : v3Version);
> > +      playerJs.src = OOYALA_PLAYER_URL;
> > +      if (debug || v3Version) {
> > +        if (!playerJs.src.match(/\?$/)) {
> > +          playerJs.src += "?";
> > +        } else if (playerJs.src.match(/=/)) {
>
> Unlikely. Let's make this implementation simple and assume that developers
> knows his/her trade in forming a good URL.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/chromecast-sample-receiver/pull/35/files#r50929385>
> .
>


-- 
-- 
Dimitry Nasledov
dima@ooyala.com